ZAPD/ZTexture.cpp

```diff
@@ -116,6 +116,7 @@ ZTexture* ZTexture::FromPNG(string pngFilePath, TextureType texType)
 		case TextureType::GrayscaleAlpha16bpp: tex->PrepareRawDataGrayscaleAlpha16(pngFilePath); break;
 		case TextureType::Palette4bpp: tex->PrepareRawDataPalette4(pngFilePath); break;
 		case TextureType::Palette8bpp: tex->PrepareRawDataPalette8(pngFilePath); break;
+		default: throw "Format is not supported!";
```

Comment:
Throwing a `const char*` is generally a bad idea and a bad practice. It _may_ be acceptable if at least ZAPD were expecting this anywhere in code (something like `catch(const char* const)`), but it doesn't.
So, if this were throw, ZAPD would print the following line and then the mangled backtrace:
`terminate called after throwing an instance of 'char const*'`

I suggest throwing a `std::runtime_error` instead, but if you want to go fancy, you could consider something like `std::domain_error` or so.
```suggestion
		default: throw std::runtime_error("Format is not supported!");
```